utils/index.js

```diff
@@ -310,21 +310,19 @@ const getPreviousRoute = (req, routes = defaultRoutes) => {
   return prevRoute()
 }
 
-const returnToCheckAnswers = (req, res, claimYes = false, routes = defaultRoutes) => {
+const returnToCheckAnswers = (req, res, claimYes = false) => {
 
   const currentRoute = getRouteWithIndexByPath(req.path)
-  const nextRoute = routes[currentRoute.index + 1]
+  const nextRoute = defaultRoutes[currentRoute.index + 1]
   const ref = req.query.ref
   
   if(
-    ref === 'checkAnswers' && 
     'editInfo' in nextRoute &&
     'editInfo' !== 'skip' &&
     claimYes
     ) {
       return res.redirect(`${nextRoute.path}?ref=checkAnswers`)
     } else if (
-      ref === 'checkAnswers' && 
       !claimYes 
```

Comment:
This is now now true by virtue of the first check (ie, it has to be truthy for the initial check to pass)

Reply:
If we wanted to have a case where 
('editInfo' in nextRoute ||  'editInfo' === 'skip') was NOT true and we would do something else, then that second check makes sense (!claimYes), but not currently.